app/frontend/src/components/RequestForm.vue

```diff
@@ -0,0 +1,199 @@
+        <v-icon large color="red">cancel</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>
+          An error occurred while attempting to add your application.
+          <br />You can also add your application by sending an email to
+          <a
+            href="mailto:NR.CommonServiceShowcase@gov.bc.ca?subject=GETOK Registration for <acronym> - <idir>"
+          >NR.CommonServiceShowcase@gov.bc.ca</a>
+        </p>
+        <p>Please include your Acronym as well as your IDIR username in your email.</p>
+      </template>
+    </BaseDialog>
+
+    <BaseDialog v-bind:show="registerSuccess" @close-dialog="registerSuccess = false">
+      <template v-slot:icon>
+        <v-icon large color="green">check_circle_outline</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>Your Registration was sent successfully. You will be sent an email to {{ userInfo.emailAddress }} when your application has been authorized.</p>
+      </template>
+    </BaseDialog>
+  </v-container>
+</template>
+
+<script>
+import Vue from 'vue';
+import { FieldValidations } from '@/utils/constants.js';
+
+export default {
+  name: 'RequestForm',
+  data: function() {
+
+    return {
+      applicationAcronym: '',
+      applicationAcronymRules: [
+        v => !!v || 'Acronym is required',
+        v =>
+          v.length <= FieldValidations.ACRONYM_MAX_LENGTH ||
+          `Acronym must be ${FieldValidations.ACRONYM_MAX_LENGTH} characters or less`,
+        v =>
+          /^(?:[A-Z]{2,}[_]?)+[A-Z]{1,}$/g.test(v) ||
+          'Incorrect format. Hover over ? for details.'
+      ],
+      comments: '',
+      errorOccurred: false,
+      fieldValidations: FieldValidations,
+      registrationDialog: false,
+      registerSuccess: false,
+      valid: false,
+      userInfo: {
+        emailAddress: Vue.prototype.$keycloak.tokenParsed.email,
+        idir:  Vue.prototype.$keycloak.tokenParsed.preferred_username,
+        name:  Vue.prototype.$keycloak.tokenParsed.name
+      }
+    };
+  },
+  computed: {
+  },
+  methods: {
+    async cancel() {
+      this.errorOccurred = false;
+      this.registrationDialog = false;
+      this.registerSuccess = false;
+      this.$router.push({ name: 'About' });
+    },
+
+    async postRegistrationForm() {
+      this.errorOccurred = false;
+      this.registerSuccess = false;
+      if (this.$refs.form.validate()) {
+        try {
+          // removed api call until i have service layer
+          const response = false;
+
+          // const response = await ApiService.sendRegistrationEmail({
+          //   applicationAcronym: this.applicationAcronym,
+          //   comments: this.comments,
+          //   from: this.userInfo.emailAddress,
+          //   idir: this.userInfo.idir
+          // });
+
+          if (response) {
+            this.registrationDialog = false;
+            this.registerSuccess = true;
+          } else {
+            this.errorOccurred = true;
```

Comment:
Two options here:
1) (Keep try-catch) just throw a new error so it cascades to Line 186 - don't need to duplicate "error cases" when they do the same thing
2) (Drop try-catch) Change to Promise.then.catch pattern and wrap as needed (the API service will be returning raw axios responses, which are promise based, so we can leverage that instead of having to use our own try catch)

Reply:
ok. i think i have it corrected